pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -926,7 +926,8 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 
 	// Step 7: start containers in podContainerChanges.ContainersToStart.
 	for _, idx := range podContainerChanges.ContainersToStart {
-		start("container", metrics.Container, containerStartSpec(&pod.Spec.Containers[idx]))
+		err = start("container", metrics.Container, containerStartSpec(&pod.Spec.Containers[idx]))
+		utilruntime.HandleError(err)
```

Comment:
nit: this change seems duplicate with https://github.com/kubernetes/kubernetes/blob/3a26b864f4e17a6b7e0e673f583a551820a25b36/pkg/kubelet/kuberuntime/kuberuntime_manager.go#L898 inside

Reply:
yes, you are right. I will think it again. Thanks very much.